ui/app/templates/vault/cluster.hbs

```diff
@@ -3,6 +3,40 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
+{{#if this.auth.showRootTokenWarning}}
+  <Modal
+    @title="Warning"
+    @onClose={{fn (mut this.auth.showRootTokenWarning) false}}
```

Comment:
The way I read this it seems like there's no way to `cancel` is that right?

I'm wondering if we should add some sort of "abort transition" action 🤔 Since it seems confusing that clicking "x" would also continue

Reply:
Yes, there's no way to `cancel` and the only way to technically move on is to click "Confirm" or click out of the modal. The "X" button doesn't show for the modal. 

<img width="1238" alt="Screenshot 2023-10-02 at 12 49 26 PM" src="https://github.com/hashicorp/vault/assets/30884335/d1cc5e78-e32e-4d3a-872d-57e050543144">

Reply:
Okay! The Hds::Modals always show an `X` button so trying to think how this would work in that context 🤔

Reply:
@hellobontempo Looking at the [HDS::Modal doc](https://hds-website-hashicorp.vercel.app/components/modal?tab=code), it looks like there's an `isDismissDisabled` arg, and I think that _might_ hide the "X" button. I think the intention is to not have the users exit out of this modal until they click "Confirm". For our old modals, the `onClose` arg is required in order for the modals to work.

Reply:
That arg disables the `X` but doesn't hide it. The only way to dismiss the modal is to change it from  `true` to `false` so the confirm action would have to change the value of the arg back to `false` 

Here is an [example](https://github.com/hashicorp/vault/blob/ui/VAULT-17315-hds-adoption-replace-Modal/ui/lib/replication/addon/templates/mode/secondaries/add.hbs#L66) of where I implemented that arg